pkg/healthcheck/healthcheck.go

```diff
@@ -919,7 +937,7 @@ func validateDataPlanePods(pods []*pb.Pod, targetNamespace string) error {
 	return nil
 }
 
-func validateDataPlanePodReporting(pods []*pb.Pod) error {
+func validateDataPlanePodReporting(pods []*pb.Pod, proxyTarget string) error {
```

Comment:
Looks like the `proxyTarget` variable is unused. Would this be used to check for data plane reporting for a specific proxy Target?

Reply:
Good catch. No, we don't need it here because the `proxyTarget` is now used directly when querying pods. See the `getDataPlanePods`.